core/common/src/main/java/alluxio/Constants.java

```diff
@@ -92,6 +92,7 @@ public final class Constants {
   public static final long FILE_SYSTEM_MASTER_WORKER_SERVICE_VERSION = 2;
   public static final long BLOCK_MASTER_CLIENT_SERVICE_VERSION = 2;
   public static final long BLOCK_MASTER_WORKER_SERVICE_VERSION = 2;
+  public static final long BLOCK_WORKER_CLIENT_SERVICE_VERSION = 1;
```

Comment:
I actually think the version number indicates something and should conform to the same number. For example the same  `BLOCK_WORKER_CLIENT_SERVICE_VERSION ` and `BLOCK_MASTER_CLIENT_SERVICE_VERSION` suggest they are from the same Alluxio version compatibility group. So if we add a version here I prefer to just use 2.

For example, if one day we upgrade `BLOCK_MASTER_CLIENT_SERVICE_VERSION` to be 3, all versions here should be changed to 3. Not sure what others think.

Reply:
If some breaking changes are made to `BLOCK_MASTER_CLIENT_SERVICE_VERSION` but ` BLOCK_WORKER_CLIENT_SERVICE_VERSION` stays unchanged, do we still need to keep them in sync?

Reply:
in this case, should we have only one version number for all?